lib/msf/ui/console/command_dispatcher/db.rb

```diff
@@ -1889,26 +1889,50 @@ def cmd_db_disconnect(*args)
     end
   end
 
-  def cmd_db_rebuild_cache
-    unless framework.db.active
-      print_error("The database is not connected")
-      return
-    end
-
-    print_status("Purging and rebuilding the module cache in the background...")
-    framework.threads.spawn("ModuleCacheRebuild", true) do
-      framework.db.purge_all_module_details
-      framework.db.update_all_module_details
-    end
-  end
-
   def cmd_db_rebuild_cache_help
     print_line "Usage: db_rebuild_cache"
     print_line
-    print_line "Purge and rebuild the SQL module cache."
+    print_line "Rebuild the local module data store"
+    print_line
+    print_line "   OPTIONS:"
+    print_line "       -h, --help          Help"
+    print_line "       -u, --update-base   Update the base data store (developers only)"
     print_line
   end
 
+  def cmd_db_rebuild_cache(*args)
```

Comment:
Given that this is not longer a db interaction we should move and rename this command as it is not longer related to the `db` layer.

Reply:
Sure, what is your proposal? Just delete this command entirely? Do you know where this command is documented already?

Reply:
Upon reflection on the search results here: https://www.google.com/search?q=db_rebuild_cache, I'm not certain that renaming this command just because the implementation details are different is the best choice, from a community support PoV. It does still update a database, even if it's not the same kind of database as before.

The only other alternative I would think makes sense would be to entirely add a new command, and rework this one to say something to the effect it's not needed when people do hit an old tutorial and try it.

Reply:
A new command and deprecation message is a reasonable approach.  

I reviewed the results of your search link and the hits are already somewhat invalid.  Most look to be about resolving `slow search` or using `search` without a database, which is what removing from the `db` was all about.  A search based on incorrect results such as: https://www.google.com/search?q=metasploit+invalid+search+results does not really offer quick guidance to use this command either, leading me to stick with rename or removal as reasonable even from the `google results` == `community support/documentation`  point of view.

Reply:
Cool, I added a new command called 'reload_search', and since 'reload' commands tend to be more developer-focused anyway, I think it fits into that scheme nicely. I'll update the PR bits shortly to match if that makes sense.